### DISCUSSION TITLE
Using multiple DAWs with BlackHole

### DISCUSSION THREAD

**Discussion by TheMoonshiner** [ExistentialAudio/BlackHole#47] [General]
I've been playing with the idea of using BlackHole to use multiple DAWs when producing music. When Soundflower discontinued I lost the ability to incorporate GarageBand & Logic Pro within Ableton. Having the ability opened a lot of creative doors for me with a wider amount of sounds and tools at my disposal. It bypassed finicky plugin compatibilities and was a huge time saver all without having to save, reopen DAWs, and swap files in a tedious manner. It also has become a great tool in terms of creative block and not limiting yourself to one audio environment in which you may pick up habits and get too comfortable with certain 'go-to's'. So far so good in terms executing what I had in mind so thank you Existential Audio, this has been a blessing for me. It got me thinking though that other's may use BlackHole in a similar way but may have run into limitations that I haven't considered yet. Maybe we can have a thread going for limitations & feature requests with using multiple DAWs in mind?

**Comment by devinroth** [comment]
You're welcome. I love hearing how people are using Blackhole.
